packages/bson-transpilers/test/yaml/partial.yaml

```diff
@@ -75,6 +80,7 @@ tests:
           java: Decimal128
           csharp: Decimal128
           shell: NumberDecimal
+          php: new \MongoDB\BSON\Decimal128
```

Comment:
What is the purpose of this template? Is it actually constructing a new Decimal128 with no parameters?

[`Decimal128::__construct()`](https://www.php.net/manual/en/mongodb-bson-decimal128.construct.php) requires an argument.

Reply:
This is a "partial" test. That means it checks not all syntax constructions, but only a part of them, like how correctly will be converted a class name.

Reply:
OK to resolve.